prebid-mobile/modules/rendering/android-sdk-integration-pb.md

```diff
@@ -2,27 +2,39 @@
 
 layout: page_v2
 title: Custom or No mediation
-description: Integration of Prebid SDK withou primaty Ad Server
+description: How to integrate the Prebid SDK without a primary Ad Server
 sidebarType: 2
 
 ---
 
+
+
 # Custom Bidding Integration
 {:.no_toc}
 
-You can use Prebid SDK to monetize your app with a custom ad server or even without it. Use the `Transport API` to obtain the targeting keywords for following usage with the custom ad server. Use the `Rendering API` to display the winning bid without primary ad server and its SDK.
+## Before you start
+
+Before implementing the instructions in this guide, you need to ensure that you have correctly initialized the Prebid Mobile SDK in your app. You can do so using the code snippet below:
+
+```
+Prebid.init(getApplicationContext(), adUnits, "INSERT-ACCOUNT-ID-HERE", Prebid.AdServer.DFP, Prebid.Host.RUBICON);
```

Comment:
We need to talk about the options for Prebid Server here. Either link to a place where this is already discussed, or something like:

"You will need to find a Prebid Server and register it with the SDK. Here are your options:
- The SDK is coded with the URLs for the following Prebid member companies: Prebid.Host.RUBICON, Prebid.Host.APPNEXUS, .... Reach out to them to discuss their [managed Prebid Server services](https://prebid.org/managed-services/).
- Other Prebid Server managed services exist that don't have enums linking to the URL. You can work with them by entering a URL into the call to init().
- You can spin up your own Prebid Server and enter the URL into the call to init().